n_t_with_s/plots.py

```diff
@@ -0,0 +1,230 @@
+"""
+Code for generating plots.
```

Comment:
it would be great to edit the figure titles so we know they are generated from models with selection

Reply:
Did it in this PR